docs/fsharp/style-guide/formatting.md

```diff
@@ -1667,7 +1683,7 @@ For any custom operator that starts with `*` and that has more than one characte
 
 ### Formatting record declarations
 
-For record declarations, by default you should indent `{` in type definition by four spaces, start the field list on the same line and align any members with the `{` token:
+For record declarations, by default you should indent `{` in type definitions by four spaces, start the field list on the same line, and align any members with the `{` token:
 
 ```fsharp
 // ✔️ OK
```

Comment:
Should line 1696 say "...with **labels** indented by an additional four spaces"?

Reply:
No, it's referring to additional members as described [here](https://learn.microsoft.com/en-us/dotnet/fsharp/language-reference/records#records-and-members). I'll update the example to include one for clarity